README.md

```diff
@@ -185,5 +185,34 @@ Create an ISO for a medium RHEL 7 system named foobar and pass a couple of extra
 $ ./mkvm.rb -t medium -r 7 --extra "console=ttyS0 ks.sendmac noverifyssl sshd" foobar
 ```
 
+## Removing servers from Puppet with rmvm.rb
+To remove a server from Puppet, you will need a certificate/key pair that is signed by your Puppet CA and allowed access to the ```certificate status``` end point. Access to the ```certificate status``` end point is configured in your Puppet server's auth.conf:
```

Comment:
I would remove this section of the documentation.  If we want to document rmvm in a more meaningful way it should be a separate MD file.